util/exercise/src/lib.rs

```diff
@@ -1,13 +1,12 @@
 #[macro_use]
 extern crate failure;
-extern crate failure_derive;
-extern crate flate2;
+
 #[macro_use]
 extern crate lazy_static;
```

Comment:
I believe that the `#[macro_use]` idiom is now deprecated, and the correct idiom should now be 

```rust
use lazy_static::lazy_static;
```

This won't be touched by autofixers, because there's no straightforward way for them to enumerate the list of macros imported by this code from that crate. 

This applies also to the `failure` crate above, and possibly others which didn't happen to show up in the diff I'm seeing.